app/services/view/AutoCompleteItem.scala

```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
-package services.model
+package services.view
```

Comment:
If we're moving this, i'd suggest it goes into the `views.components.inputs` package as its only relevant to the `input_radio` and stuff that uses that

Reply:
services.views => views is usually only for .html.scala files
services.model is needed for the backend CSV parsing

Was discussed with Matt / Pat and agreed services.view ....

Reply:
"views is usually only for .html.scala files" Just my view but anything view related should go in the `views` package, that includes AutoComplete, RadioOption and whatever other view helpers we have

Reply:
At the moment we're using the `services` package rather like a `utils` package